CL/cl_ext.h

```diff
@@ -2441,6 +2441,15 @@ typedef struct _cl_queue_family_properties_intel {
 #define CL_QUEUE_CAPABILITY_BARRIER_INTEL                   (1 << 25)
 #define CL_QUEUE_CAPABILITY_KERNEL_INTEL                    (1 << 26)
 
+/***************************************************************
+* cl_intel_queue_no_sync_operations
+***************************************************************/
+
+#define cl_intel_queue_no_sync_operations 1
+
+/* addition to cl_command_queue_properties */
+#define CL_QUEUE_NO_SYNC_OPERATIONS_INTEL                   (1 << 27)    
```

Comment:
why did you go for `27` out of interest? Wondering if there are other extensions that haven't reserved the xml value. `29` is the highest currently available bit position in https://github.com/KhronosGroup/OpenCL-Docs/blob/main/xml/cl.xml#L793

Reply:
Yes, I got confused by CL_QUEUE_CAPABILITY_KERNEL_INTEL. I'll change it to 29.